src/main/java/io/appium/java_client/ios/IOSMobileCommandHelper.java

```diff
@@ -81,4 +81,15 @@
         return new AbstractMap.SimpleEntry<>(
                 SHAKE, ImmutableMap.<String, Object>of());
     }
+    
+    /**
+     * This method forms a {@link java.util.Map} of parameters for the touchId simulator
+     * 
+     * @param match Are we simulating a successful fingerprint scan?
```

Comment:
It's how I've documented boolean parameters in the past. I'll change it to be more in line with how it's normally documented.